From: Felipe Sateler <fsateler@debian.org>
Date: Fri, 25 Oct 2019 08:39:03 -0300
Subject: qpaeq: use python3 instead of python 2

The script was already compatible, so lets just switch interpreter
Forwarded: https://gitlab.freedesktop.org/pulseaudio/pulseaudio/merge_requests/197
---
 src/utils/qpaeq | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/utils/qpaeq b/src/utils/qpaeq
index 914aedf..7e01d87 100755
--- a/src/utils/qpaeq
+++ b/src/utils/qpaeq
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #    qpaeq is a equalizer interface for pulseaudio's equalizer sinks
 #    Copyright (C) 2009  Jason Newton <nevion@gmail.com
 #
